dipy/tracking/localtrack.pyx

```diff
@@ -129,70 +129,23 @@ def local_tracker(
     cdef:
         cnp.npy_intp i
         StreamlineStatus stream_status
-        double dir[3]
-        double vs[3]
-        double seed[3]
+        double use_fixed_step
 
     if (seed_pos.shape[0] != 3 or first_step.shape[0] != 3 or
             voxel_size.shape[0] != 3 or streamline.shape[1] != 3):
         raise ValueError('Invalid input parameter dimensions.')
 
-    for i in range(3):
-        dir[i] = first_step[i]
-        vs[i] = voxel_size[i]
-        seed[i] = seed_pos[i]
-
-    i = _local_tracker(dg, sc, seed, dir, vs, streamline,
-                       step_size, fixedstep, &stream_status)
-    return i, stream_status
-
-
-@cython.boundscheck(False)
-@cython.wraparound(False)
-@cython.cdivision(True)
-cdef int _local_tracker(DirectionGetter dg,
-                        StoppingCriterion sc,
-                        double* seed,
-                        double* dir,
-                        double* voxel_size,
-                        cnp.float_t[:, :] streamline,
-                        double step_size,
-                        int fixedstep,
-                        StreamlineStatus* stream_status):
-    cdef:
-        cnp.npy_intp i
-        cnp.npy_intp len_streamlines = streamline.shape[0]
-        double point[3]
-        double voxdir[3]
-        void (*step)(double*, double*, double) nogil
-
     if fixedstep:
-        step = fixed_step
+        use_fixed_step = 1
     else:
-        step = step_to_boundary
+        use_fixed_step = 0
 
```

Comment:
`use_fixed_step` I do not think this variable is necessary. 

you can use `fixedstep` variable above.

- if `fixedstep` is a bool, you can do `int(fixedstep)` to get your `int`. 
-  if `fixedstep` is a int, you already have your value.

so `use_fixed_step` is a duplicate.

Reply:
yes... forgot to clean this. thx.